py/server/tests/test_barrage.py

```diff
@@ -59,7 +59,7 @@ def tearDown(self) -> None:
         self.pub_session.close()
 
     def test_barrage_session(self):
-        session = barrage_session(host="localhost", port=10000, auth_type="Anonymous")
+        session = barrage_session(host="localhost", port=10000, auth_type="Anonymous", extra_headers={"envoy-prefix": "test"})
```

Comment:
Interesting choice of the header name, I thought in a cluster deployment, a server-server connection doesn't involve envoy.

Reply:
I'd also like to add a test method such as `def test_barrage_session_with_extra_headers` instead of modifying the existing one.

Reply:
The intended and tested use case (in Core+) is client-server where "client" is outside of "server"'s cluster. Feel free to change header to whatever is better suited for Core tests. I just chose what was familiar.

Reply:
I moved the change to its own test as suggested.